the-collab-lab/tcl-32-smart-shopping-list
# Jr mh read write firestore

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

This PR adds a single component "ReadWriteFirestore.js" that renders a button and all information currently in the collection. The button when clicked, adds +1 to the state and adds the current state to the collection. The useEffect will then listen for updates on the collection and when updated, will map over the collection and display all values. 

## Related Issue

Closes #1 

## Acceptance Criteria


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before


![](https://cdn.zappy.app/1cb786a22e820549fd843957b3e948ba.png)

### After

![](https://cdn.zappy.app/14f53306c6c20ed8e01bc07a83009c89.png)


## Testing Steps / QA Criteria

1. Open project and new terminal 
2. npm i
2. npm start to run server
3. Go to browser http://localhost:3000/
3. Click the button and see the collection render out
